gtsfm/averaging/rotation/shonan.py

```diff
@@ -58,23 +65,27 @@ def __get_shonan_params(self) -> ShonanAveragingParameters3:
         return shonan_params
 
     def __between_factors_from_2view_relative_rotations(
-        self, i2Ri1_dict: Dict[Tuple[int, int], Rot3], old_to_new_idxs: Dict[int, int]
-    ) -> BetweenFactorPose3s:
+        self,
```

Comment:
nit: maybe change method name since it doesnt return between factors anymore?

Reply:
Updated the name, thanks.